interface/patient_file/summary/add_edit_issue.php

```diff
@@ -840,5 +841,22 @@ function divclick(cb, divid) {
  Calendar.setup({inputField:"form_end", ifFormat:"%Y-%m-%d", button:"img_end"});
  Calendar.setup({inputField:"form_return", ifFormat:"%Y-%m-%d", button:"img_return"});
 </script>
+
+<!--Page Form Validations-->
+<?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
```

Comment:
Sure.
<?php if ($collectThis):?>
            <!--//include new rules of submitme functionallity-->
            <?php require_once($GLOBALS['srcdir']."/validation/validation_script.js.php");?>
.
.
.
?>